src/components/Counter/Counter.jsx

```diff
@@ -1,11 +1,55 @@
-import { useEffect, useState } from 'react'
+import { useEffect, useState, useReducer } from 'react'
 
 const pinkRGB = `rgb(236, 72, 153)`
 
 export default function Counter() {
-  const [count, setCount] = useState(0)
+  const initialCount = 0
```

Comment:
the initial value traditionally hangs out before the view or component function, like on line 4